release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1058,7 +1056,7 @@ With this update, the image change trigger controller continuously checks the co
 [id="ocp-4-10-jenkins-bug-fixes"]
 ==== Jenkins
 
-* Before this update, version 1.0.48 of the OpenShift Jenkins Sync Plugin introduced a `NullPointerException` error when Jenkins notified the plug-in of new jobs that were not associated with an OpenShift Jenkins Pipeline Build Strategy Build Config. Ultimately, this error was benign because there was no `BuildConfig` object to associate with the incoming Jenkins Job. Core Jenkins ignored the exception in our plug-in and moved on to the next listener. However, a long stack trace showed up in the Jenkins log that distracted users. With this update, the plug-in resolves the issue by making the proper checks to avoid this error and the subsequent stack trace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2030692[BZ#2030692])
+* Before this update, version 1.0.48 of the OpenShift Jenkins Sync Plug-in introduced a `NullPointerException` error when Jenkins notified the plug-in of new jobs that were not associated with an OpenShift Jenkins Pipeline Build Strategy Build Configuration. Ultimately, this error was benign because there was no `BuildConfig` object to associate with the incoming Jenkins Job. Core Jenkins ignored the exception in our plug-in and moved on to the next listener. However, a long stack trace showed up in the Jenkins log that distracted users. With this update, the plug-in resolves the issue by making the proper checks to avoid this error and the subsequent stack trace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2030692[BZ#2030692])
```

Comment:
You're right according to the ISG, but since the plug-in appears to be called the [OpenShift Jenkins Sync Plugin](https://docs.openshift.com/container-platform/4.9/cicd/builds/build-strategies.html#builds-understanding-openshift-pipeline_build-strategies-s2i), it's better to be consistent with the existing name. (Or open a bug to change it everywhere.) I don't see other use of the build configuration, though, so your change there is consistent with the [glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#build-configuration).